### DISCUSSION TITLE
What might XState editor (stately.ai studio) build?

### DISCUSSION THREAD

**Discussion by linonetwo** [retejs/rete#610] [General]
Can I use retejs build something simillar? https://stately.ai/registry/new?mode=Design

![截屏2023-04-12 17 08 35](https://user-images.githubusercontent.com/3746270/231410133-7a5d282c-779f-472a-a7d8-626adc0a543d.png)

**Comment by Ni55aN** [comment]
Yes, sure. You can find the necessary elements in the examples: https://retejs.org/examples

In general, you need examples of customization for the framework you are using ([react](https://retejs.org/examples/react/customization/), [vue](https://retejs.org/examples/vue/customization/), [angular](https://retejs.org/examples/angular/customization/)), [comment plugin](https://retejs.org/examples/comments/) or [scopes plugin](https://retejs.org/examples/scopes) (to make the frame), as well as the example of [undirected graph](https://retejs.org/examples/undirected/) to have no explicit ports
